### ISSUE TITLE
Bot jerks around when surrounded by bigger cells

### ISSUE THREAD

**Issue by ghost** [Apostolique/Agar.io-bot#269]
Title says it all. Also, when it jerks around, sometimes it will run directly into another cell.

**Comment by Timtech** [comment]
Yep, I even had my bot run directly into another cell from a good distance. Hard distance needs to be added in to the factor - if you have a 9x as large cell at you and there's one barely big enough to eat you, they are both still threats...

**Comment by FlareHAX0R** [comment]
I've had to take manual control of my bot to run out of stuff like this.

**Comment by Crayder** [comment]
If your cell is surrounded it won't know which way to go. Also, it can actually give up eventually and just continue in it's last direction. This isn't a problem for me, I only turn the bot on at night so it can level me up at night.

**Comment by avarices** [comment]
I also experience this. Sometimes I do not have enough time to take manual control, so it can be very annoying.

**Comment by Apostolique** [comment]
I already figured out how to fix this, but I won't have time to work on anything for a couple weeks probably.
